src/services/api.js

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios'
+
+const API_BASE_URL =
+  import.meta.env.VITE_APP_API_BASE_URL ||
+  window?.config?.API_BASE_URL.replace?.(/\/$/, '')
+
+export const getIndexedTariffPrices = async ({ tariff, geoZone }) => {
+  return axios({
+    method: 'GET',
+    url: `${API_BASE_URL}/data/indexed_prices`,
+    withCredentials: false,
+    params: {
```

Comment:
Reviewing the ERP code we saw that some params are missing:

- `first_date`
- `last_date`

Here the related code:
- https://github.com/Som-Energia/openerp_som_addons/blob/ffc522b7e2a2b605faa537457a434f04bcdc32df/www_som/som_indexada_webforms_helpers.py#L169
- https://github.com/Som-Energia/openerp_som_addons/blob/ffc522b7e2a2b605faa537457a434f04bcdc32df/www_som/som_indexada_webforms_helpers.py#L205C50-L205C59